docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -5,71 +5,87 @@ sidebar_label: Managing credentials
 
 # Managing Credentials
 
-To manage the progression of changes from stage to stage, Kargo will
-often require read/write permissions on private GitOps repositories and
-read-only permissions on private container image and/or Helm chart repositories.
+To manage the progression of freight from stage to stage, Kargo will often
+require read/write permissions on private GitOps repositories and read-only
+permissions on private container image and/or Helm chart repositories.
 
 This section presents an overview of how these credentials can be managed.
 
 ## Credentials as Kubernetes `Secret` Resources
 
-Kargo borrows its general credential-management approach from Argo CD, meaning
-that credentials are stored as Kubernetes `Secret` resources containing
-specially-formatted data. These secrets generally take the following form:
+:::caution
+Kargo formerly borrowed its general credential-management approach from Argo CD,
+but has since diverged.
+:::
+
+Kargo expects any credentials it requires to have been stored as specially
+labeled Kubernetes `Secret` resources containing specially-formatted data. These
+`Secret`s take the following form:
 
 ```yaml
 apiVersion: v1
 kind: Secret
 metadata:
   name: <name>
-  namespace: <namespace>
+  namespace: <project namespace>
   labels:
-    kargo.akuity.io/secret-type: <secret type>
+    kargo.akuity.io/cred-type: <cred type>
 stringData:
-  type: <type>
-  url: <repo url>
+  repoURL: <repo url>
+  repoURLPattern: <a regular expression that may match many similar repo urls>
   username: <username>
   password: <password>
 ```
 
-The `name` of such a secret is inconsequential and may follow any convention
+The `name` of such a `Secret` is inconsequential and may follow any convention
 preferred by the user.
 
 :::info
-Kargo uses Kubernetes `Namespace`s to demarcate project boundaries, so related
-`Stage` resources always share a `Namespace`.
-
-Secrets representing credentials will typically exist in the same `Namespace` as
-the `Stage` resources that will require them. There are exceptions to this rule,
-which are covered in the next section.
+Kargo uses Kubernetes `Namespace`s to demarcate project boundaries. `Secret`s
```

Comment:
Nit: when writing documentation, it is often best to use simple English (as much as possible) to help non-native speakers grasp things easily.

I think "demarcate" may be a word some people will have a harder time to understand, and I would suggest to change it to something like "define" or "mark off".

Reply:
Agree, and I'm usually more mindful of that and this is a place where I probably overestimated familiarity with that word. Happy to fix this.

Reply:
Fixed in 08c28a8895c3c3af034e9b6eb125241bc255add4